syntax = "proto3";

package adlik.serving;
option cc_enable_arenas = true;

message SSLConfig {
  string server_key = 1;
  string server_cert = 2;
  string custom_ca = 3;
  bool client_verify = 4;
};
